app/views/posts/index.html.erb

```diff
@@ -1,8 +1,22 @@
 <% @posts.each do |post| %>
   <% user = User.where(id: post.user_id) %>
   <br>
-  <%= post.content %> <br>
   <% user.each do |user| %>
-    User: <%= user.username %>
-  <% end %>
+    Post Author: <%= user.username %>
+  <% end %><br>
+   Reactions:
+  <%= post.likes %><br><br>
+  Content: <br>  <%= post.content %> <br><br>
+
+  <% post.comments.each do |comment| %>
+  comment author: <%= User.where(id: comment.author_id).pluck(:username).first %><br>
```

Comment:
- [ ] **[REQUIRED]** Try to keep queries in the models (users model in this case), you can style it as a scope or an association for example.